### ISSUE TITLE
Does not support .avi-files

### ISSUE THREAD

**Issue by filipallberg** [fortana-co/react-dropzone-uploader#44]
I've noticed that the component does not work when uploading .avi-files. The following is outputted to the console,

```
HTTP “Content-Type” of “video/x-msvideo” is not supported. Load of media resource blob:...
Cannot play media. No decoders for requested formats: video/x-msvideo
```

I assume this happens as a result of an attempt to render a preview for the file.
Secondly, it is not possible to remove the offending uploaded file.

Is it possible to turn off the thumbnail preview functionality?

Here is a sample file which cannot be uploaded: [Megamind.avi](https://github.com/opencv/opencv/blob/master/samples/data/Megamind.avi)

The problem is reproducible in the live examples hosted here: https://react-dropzone-uploader.js.org/docs/examples Attempting to upload [Megamind.avi](https://github.com/opencv/opencv/blob/master/samples/data/Megamind.avi) will fail.

---

I'm not the only one suffering from this,
https://stackoverflow.com/questions/57948791/react-dropzone-uploader-avi-file-stuck-on-status-preparing

**Comment by kylebebak** [comment]
Hey there, thanks for filing this bug report. I hadn't seen an AVI file in a long time.

I just pushed a fix in `2.10.2` that ensures that the `generatePreview` returns a promise that never takes more than 1s to resolve or reject.

This way file types for which `onload` or `onloadedmetadata` are never called don't return promises from this function that hang forever.

1s is still more than enough time to get the metadata for even huge (~5gb files), but isn't so long that it's a huge annoyance for file types that don't support metadata retrieval.

Please let me know if this fixes the problem. If it doesn't I'll reopen!

**Comment by filipallberg** [comment]
Perfect, it works!
